persistence/build.gradle

```diff
@@ -76,6 +78,11 @@ android {
         androidTest.assets.srcDirs += files("$projectDir/schemas".toString())
     }
 
+    lintOptions {
+        // To ignore false-positive lint errors
+        baseline file("lint-baseline.xml")
+    }
```

Comment:
Lint does not notice that we're using NIO via the desugaring which allows us to use NIO with minSdk 21 instead of 26.
Thus, I added a baseline file which makes lint ignore these specific errors.